### PR TITLE
fixed rbf kernel in cka

### PR DESCRIPTION
**Pull request by LukasMut**
fixed sigma arg in rbf kernel

### CODE REVIEW

**Review by a1247418 — APPROVED** [review]
LGTM

File: thingsvision/core/cka/cka_numpy.py [hunk 1]
```diff
@@ -59,13 +59,15 @@ def linear_kernel(self, X: Array) -> Array:
...
 
-    def rbf_kernel(self, X: Array, sigma: float = 1.0) -> Array:
+    def rbf_kernel(self, X: Array) -> Array:
         """Use an rbf kernel for computing the gram matrix. Sigma defines the width."""
         GX = X @ X.T
         KX = np.diag(GX) - GX + (np.diag(GX) - GX).T
-        if sigma is None:
+        if self.sigma is None:
             mdist = np.median(KX[KX != 0])
             sigma = math.sqrt(mdist)
```
Comment (lciernik) [inline thingsvision/core/cka/cka_numpy.py:68]: Question, if self.sigma=None, you compute it's value, but shall it be stored? Or shall we recompute it every time?

**Review by lciernik — COMMENTED** [review]
Left one question otherwise LGTM

File: thingsvision/core/cka/cka_numpy.py (hunk 1 above)
Comment (lciernik) [reply thingsvision/core/cka/cka_numpy.py:68]: Ok, agree

**Review by lciernik — APPROVED** [review]

**Pull request merged**